specs/signature-specification.md

```diff
@@ -40,21 +40,65 @@ Signature Manifest Example
+    - `application/cose`
+- **`subject`** (*descriptor*): A REQUIRED artifact descriptor referencing the signed manifest, including, but not limited to image manifest, image index, OCI artifact manifest.
 - **`annotations`** (*string-string map*): This REQUIRED property contains metadata for the artifact manifest.
   It is being used to store information about the signature.
   Keys using the `io.cncf.notary` namespace are reserved for use in Notary and MUST NOT be used by other specifications.
   - **`io.cncf.notary.x509chain.thumbprint#S256`**: A REQUIRED annotation whose value contains the list of SHA-256 fingerprint of signing certificate and certificate chain (including root) used for signature generation. The annotation name contains the hash algorithm as a suffix (`#S256`) and can be extended to support other hashing algorithms in future.
     The list of fingerprints is present as a JSON array string, corresponding to ordered certificates in [*Certificate Chain* unsigned attribute](#unsigned-attributes) in the signature envelope.
 
+Notary v2 supports using [OCI image manifest][oci-image-manifest] to store the signature in the repository that doesn't support [OCI artifact manifest][oci-artifact-manifest].
+
+Signature Manifest Example per OCI image manifest
+
+```jsonc
+{
+    "schemaVersion": 2,
+    "mediaType": "application/vnd.oci.artifact.manifest.v1+json",
+    "config": {
+      "mediaType": "application/vnd.cncf.notary.signature",
+      "size": 2,
+      "digest": "sha256:b5b2b2c507a0944348e0303114d8d93aaaa081732b86451d9bce1f432a537bc7"
+    },
+    "layers": [
+        {
+            "mediaType": "application/jose+json",
+            "digest": "sha256:9834876dcfb05cb167a5c24953eba58c4ac89b1adf57f28f2f9d09af107ee8f0",
+            "size": 32654
+        }
+    ],
+    "subject": {
+        "mediaType": "application/vnd.oci.image.manifest.v1+json",
+        "digest": "sha256:73c803930ea3ba1e54bc25c2bdc53edd0284c62ed651fe7b00369da519a3c333",
+        "size": 16724
+    },
+    "annotations": {
+        "io.cncf.notary.x509chain.thumbprint#S256": 
+        "[\"B7A69A70992AE4F9FF103EBE04A2C3BA6C777E439253CE36562E6E98375068C3\",\"932EB6F5598435D4EF23F97B0B5ACB515FAE2B8D8FAC046AB813DDC419DD5E89\"]"
+    }
+}
+```
+
+- **`config`** (*array of objects*): This descriptor is REQUIRED by OCI image manifest. No config object is required by Notary v2 signatures.
+  - **`mediaType`** (*string*): This REQUIRED property contains the Notary version of the signature: `application/vnd.cncf.notary.signature`. The value is equivalent to the value used in the `artifactType` in OCI artifact manifest.
+  - **`digest`** (*string*): This REQUIRED property contains dummy data.
+  - **`size`** (*int64*): This REQUIRED property contains dummy data.
+- **`layers`** (*array of objects*): This REQUIRED property contains collection of only one [OCI descriptor][oci-descriptor] referencing signature envelope.
```

Comment:
How will we support multiple signatures for an image?

Reply:
Every signature is stored as one OCI artifact manifest, and under the layers section of OCI artifact manifest, the digest points to the `real` signature (JWS or COSE). Under the subject area of OCI artifact manifest, the digest points to the image that the signature is associated with.

It is not like that "all the signatures are referenced under the layers section of one OCI artifact manifest"